core/src/main/scala/partialEvaluator/PartialEvaluator.scala

```diff
@@ -0,0 +1,250 @@
+package stainlessfit
+package core
+package partialEvaluator
+
+import stainlessfit.core.extraction._
+import java.io.File
+import java.io.BufferedWriter
+import java.io.FileWriter
+import smtlib.printer.Printer
+import trees._
+import util.RunContext
+import TreeUtils._
+import parser.FitParser
+import stainlessfit.core.util.Utils
+import interpreter.Interpreter
+import scala.io.StdIn._
+
+object PartialEvaluator {
+  def pipeline()(implicit rc: RunContext) =
+        DebugPhase(new DefFunctionElimination(), "DefFunctionElimination") andThen
+        DebugPhase(new Namer(), "Namer") andThen
+        DebugPhase(new BuiltInIdentifiers(), "BuiltInIdentifiers") andThen
+        DebugPhase(new Erasure(), "Erasure")
+
+
+
+  val zero = BigInt(0)
+  def subError(a: BigInt,b: BigInt) = Error(s"Subtraction between ${a} and ${b} will yield a negative value", None)
+  def divError = Error(s"Attempt to divide by zero", None)
+
+  val __ignoreRefCounting__ = true
+  val __ignoreMeasure__ = true
+  val __debugPrint__ = false
```

Comment:
those debug values should be exposed to the command line

Reply:
Yes please.